ng-dev/pr/common/validation/validations.ts

```diff
@@ -28,7 +28,17 @@ export function assertChangesAllowForTargetLabel(
   label: TargetLabel,
   config: PullRequestConfig,
   releaseTrains: ActiveReleaseTrains,
+  labelsOnPullRequest: string[],
 ) {
+  if (
+    !!config.commitMessageFixupLabel &&
+    labelsOnPullRequest.some((name) => matchesPattern(name, config.commitMessageFixupLabel))
+  ) {
+    debug('Skipping assertion for changes from commits allowed for the target label because the');
```

Comment:
Optional nit: This seems to read a little confusingly, maybe because I'm not a native-speaker. I think something like the following would be more clear:

```
Skipping commit message target label validation because the commit message fixup label is applied.
```